distributed/dashboard/components/shared.py

```diff
@@ -21,14 +21,14 @@
 
 import dask
 
-from distributed import profile
-from distributed.compatibility import WINDOWS
-from distributed.dashboard.components import DashboardComponent
-from distributed.dashboard.utils import update
-from distributed.utils import log_errors
+from ... import profile
+from ...compatibility import WINDOWS
+from ...utils import log_errors
+from ..utils import update
```

Comment:
This is a nice example how things can become quite ambiguous

Reply:
Agreed, this is just the sort of thing I'd like to avoid having to read.